### PR TITLE
Import utilities from plugin-utilities repo

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

File: kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt [hunk 1]
```diff
@@ -2,9 +2,9 @@ package org.jetbrains.research.ml.kotlinAnalysis
 
 import com.intellij.psi.util.PsiTreeUtil
 import org.jetbrains.kotlin.psi.KtNamedFunction
-import org.jetbrains.research.ml.kotlinAnalysis.util.Extension
-import org.jetbrains.research.ml.kotlinAnalysis.util.ParametrizedBaseTest
-import org.jetbrains.research.ml.kotlinAnalysis.util.getPsiFile
+import org.jetbrains.research.pluginUtilities.util.Extension
```
Comment (tiginamaria) [inline kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt:5]: Maybe we can either use import `org.jetbrains.research.pluginUtilities.util.Extension` in our package `org.jetbrains.research.ml.kotlinAnalysis.util` or move `VirtualFile.isKotlinRelatedFile()` there?

**Review by tiginamaria — COMMENTED** [review]
Great! Thank you! We missed this very much!

File: kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt (hunk 1 above)
Comment (nbirillo) [reply kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt:5]: What about the third option: use `org.jetbrains.research.pluginUtilities.util.Extension` in `isKotlinRelatedFile` function?) and delete `Extension` class from our repo

Comment (tiginamaria) [reply kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt:5]: Yes, sure! I failed to explain it, but this was my idea =)

**Review by tiginamaria — APPROVED** [review]
Perfect!

File: kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt (hunk 1 above)
Comment (annav1asova) [reply kotlin-analysis-clones/src/test/kotlin/org/jetbrains/research/ml/kotlinAnalysis/CloneDetectionAdapterTest.kt:5]: I used in range analysis Extension.CSV, can you add it to utilities either, please?

**Review by annav1asova — APPROVED** [review]
Super!

**Pull request merged**